File: src/main/java/com/ptah/camunda/delegate/SayHelloDelegate.java
package com.ptah.camunda.delegate;

import org.camunda.bpm.engine.delegate.DelegateExecution;
import org.camunda.bpm.engine.delegate.JavaDelegate;
import org.springframework.stereotype.Component;

@Component("SayHelloDelegate")
public class SayHelloDelegate implements JavaDelegate {
    @Override
    public void execute(DelegateExecution execution) throws Exception {
        execution.setVariable("output",execution.getVariable("flow"));
    }
}

File: backend/src/main/kotlin/com/gihub/jarryzhou/ptah/camunda/delegate/SayHelloDelegate.kt
package com.gihub.jarryzhou.ptah.camunda.delegate

import org.camunda.bpm.engine.delegate.DelegateExecution
import org.camunda.bpm.engine.delegate.JavaDelegate
import org.springframework.stereotype.Component

@Component("SayHelloDelegate")
class SayHelloDelegate : JavaDelegate {
    @Throws(Exception::class)
    override fun execute(execution: DelegateExecution) {
        execution.setVariable("output", execution.getVariable("flow"))
    }
}
